pubtools/_quay/signature_handler.py

```diff
@@ -431,7 +432,19 @@ def construct_item_claim_messages(self, push_item):
         claim_messages = []
 
         if push_item.claims_signing_key:
-            digests = self.get_tagged_image_digests(push_item.metadata["pull_url"])
+            if (
+                QuayClient.MANIFEST_LIST_TYPE
+                in push_item.metadata["build"]["extra"]["image"]["media_types"]
+            ):
+                mtype = None
```

Comment:
I'd rather you uniformly support get_manifest with all types instead of inexplicably using "None" here

Reply:
None means preferably manifest list, but in this case I think it can be changed to list directly